UnityProject/Assets/Scripts/Util/ConverterExtensions.cs

```diff
@@ -337,58 +153,30 @@ public static Vector2 GetRandomRotatedVector2(float minimum, float maximum)
 		       GetRandomNumber(minimum, maximum); //the * Minus number will do the other side Making it full 360
 	}
 
-	public static Vector2Int ToLocalVector2Int(this OrientationEnum In)
-	{
-		return ToLocalVector3(In).To2Int();
-	}
-
-	public static Vector3 ToLocalVector3(this OrientationEnum In)
+	public static Vector2Int ToLocalVector2Int(this OrientationEnum @in)
 	{
-		switch (In)
-		{
-			case OrientationEnum.Up_By0:
-				return Vector3.up;
-			case OrientationEnum.Right_By270:
-				return Vector3.right;
-			case OrientationEnum.Down_By180:
-				return Vector3.down;
-			case OrientationEnum.Left_By90:
-				return Vector3.left;
-
-		}
-		return Vector3.zero;
+		return ToLocalVector3(@in).RoundTo2Int();
 	}
 
-
-	public static OrientationEnum ToOrientationEnum(this Vector2Int direction)
-	{
-		if (direction == Vector2Int.down)
-		{
-			return OrientationEnum.Down_By180;
-		}
-		else if (direction == Vector2Int.left)
+	public static Vector3 ToLocalVector3(this OrientationEnum @in) =>
+		@in switch
 		{
-			return OrientationEnum.Left_By90;
-		}
-		else if (direction == Vector2Int.up)
+			OrientationEnum.Up_By0 => Vector3.up,
+			OrientationEnum.Right_By270 => Vector3.right,
+			OrientationEnum.Down_By180 => Vector3.down,
+			OrientationEnum.Left_By90 => Vector3.left,
+			_ => Vector3.zero
+		};
+
+	public static OrientationEnum ToOrientationEnum(this Vector2Int direction) =>
+		direction switch
 		{
-			return OrientationEnum.Up_By0;
-		}
-		else if (direction == Vector2Int.right)
-		{
-			return OrientationEnum.Right_By270;
-		}
-		else if (direction.y == -1)
-		{
-			return OrientationEnum.Down_By180;
-		}
-		else if (direction.y == 1)
-		{
-			return OrientationEnum.Up_By0;
-		}
-
-		return OrientationEnum.Down_By180;
-	}
-
-
+			// Based on what was checked and returned before. These are very specific and don't account for anything
+			// greater than one or one direction greater than another for example. Is it supposed to be that way?
```

Comment:
CC @Bod9001 thoughts?

Reply:
for coordinate values it would be better to use ; (semi-comma) 
(comma) it's more for sub-division

like x;y
x;y;z
x,xxx;y,xxx

Reply:
Okay, I'll add that note in a summary for that method tonight. Is it supposed to return the Down_Left, Down_Right, and so on orientations too? The original only returned the Up, Right, Down, Left orientations.